internal/ethapi/api_test.go

```diff
@@ -1244,7 +1244,7 @@ func TestFillBlobTransaction(t *testing.T) {
 			if len(tc.err) > 0 {
 				if err == nil {
 					t.Fatalf("missing error. want: %s", tc.err)
-				} else if err != nil && err.Error() != tc.err {
+				} else if err.Error() != tc.err {
```

Comment:
Why did you remove the `nil`-check?

Reply:
Because line 1245 has checked `if err==nil`, so in `else`, do not need to check `err!=nil`